cli/compilers/ts.rs

```diff
@@ -287,13 +282,13 @@ impl TsCompiler {
       "Invoking the compiler to bundle. module_name: {}",
       module_name
     );
+    eprintln!("Bundling {}", module_name);
```

Comment:
Is this exposed to users? Maybe make this debug! or maybe use the green bold we use for compiling?

Reply:
Yeah, it was printed using `console.log` in TS worker - IMO we should remove it altogether along with `Emmited X bytes` message